compatibility/bazel_tools/testing.bzl

```diff
@@ -274,14 +274,16 @@ excluded_test_tool_tests = [
         ],
     },
     {
+        # We drop visibily restrictions about local contract key in
+        #  https://github.com/digital-asset/daml/pull/15131
+        # Since this change does not raise any obvious security problems, it
+        # seems superfluous to continue checking that old SDKs enforce the
+        # restriction. So we disable completely the test.
         "start": "1.13.0-snapshot.20210419.6730.1.8c3a8c04",
+        "end": "2.6.0-snapshot.20221226.11190.0.71548477",
```

Comment:
Don't you want an open end here? Otherwise ledger API test tool vresions that are newer than the `end` you set here will check it again.

Reply:
The test is gone in main.

Reply:
ah I see, makes sense thx